warehouse/query-core/src/main/java/datawave/query/ancestor/AncestorQueryIterator.java

```diff
@@ -311,4 +312,11 @@ public Equality getEquality() {
         }
         return equality;
     }
+
+    @Override
+    protected DefaultOptions createDefaultOptions() {
```

Comment:
We do not need to override createDefaultOptions() here, because a different default value for the HIT_LIST options is never actually established. All changes to this class can be reverted.